README.md

```diff
@@ -58,180 +29,127 @@ After you're finished please remove all the comments and instructions!
+In this project best coding practices is use
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
 To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
 In order to run this project you need:
 
-<!--
-Example command:
+working in browser
+
+```sh
+  
+```
+
+Github Account:
+```sh
+  https://github.com/
+```
+Npm installed: 
+```sh
+  https://nodejs.org/en/download
+```
 
+Git installed: 
 ```sh
- gem install rails
+  https://git-scm.com/book/en/v2/Getting-Started-Installing-Git
 ```
- -->
 
 ### Setup
 
 Clone this repository to your desired folder:
 
-<!--
-Example commands:
-
 ```sh
   cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+  git clone github.com/Alibaba2023/Hello-Microverse.git
 ```
---->
 
 ### Install
 
 Install this project with:
 
-<!--
-Example command:
+Linters:
 
+Lighthouse run:
 ```sh
-  cd my-project
-  gem install
+  npm install -g @lhci/cli@0.7.x
+```
+Webhint run 
+```sh
+ npm install --save-dev hint@7.x
+```
+StyleLint run:
+```sh
+ npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
 ```
---->
 
 ### Usage
 
 To run the project, execute the following command:
 
-<!--
-Example command:
-
 ```sh
-  rails server
+  cd HelloMicroverse
+  index.html
 ```
---->
 
 ### Run tests
 
 To run tests, run the following command:
 
-<!--
-Example command:
+to check for styling errors:
 
 ```sh
-  bin/rails test test/models/article_test.rb
+  npx stylelint "**/*.{css,scss}"
+```
+To test optimization:
+```sh
+  lighthouse [site url]
 ```
---->
 
 ### Deployment
 
 You can deploy this project using:
 
-<!--
-Example:
-
-```sh
-
-```
- -->
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- AUTHORS -->
+Your working browser.
 
 ## 👥 Authors <a name="authors"></a>
 
-> Mention all of the collaborators of this project.
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- FUTURE FEATURES -->
-
-## 🔭 Future Features <a name="future-features"></a>
-
-> Describe 1 - 3 features you will add to the project.
-
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+👤 **Ali Baba Hussaini**
 
-<!-- CONTRIBUTING -->
+- GitHub: [@githubhandle](https://github.com/Alibaba2023)
+- Twitter: [@twitterhandle](https://twitter.com/Alibaba)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ali-baba-hussaini-630607267/)
```

Comment:
- [ ] [OPTIONAL] Great job adding the links to your profiles however I suggest using your own handles on those author links instead of default `@githubhandle` , `@twitterhandle `and `LinkedIn`